packages/kit/src/runtime/client/client.js

```diff
@@ -1217,19 +1217,21 @@ export function create_client({ target, base, trailing_slash }) {
 			return goto(href, opts, []);
 		},
 
-		invalidate: (resource) => {
-			if (resource === undefined) {
+		invalidate: (...resources) => {
```

Comment:
With that we lose the ability to add an options object down the road, which was IIRC was one of the reasons for spreading out `invalidate` and `invalidateAll` - are we ok with that?

Reply:
excellent point. let's revisit later if enough time goes by that we're confident we won't need to add options. i guess another option would be to allow arrays, but that would be at odds with `depends(...urls)` and generally less nice.

will add a note to the issue